### ISSUE TITLE
Import Express Forms page not loading

### ISSUE THREAD

**Issue by jmcgaha** [solspace/craft3-freeform#1287]
### What happened?

Go to Freeform -> Import / Export -> Express Forms
The page refreshes and no content appears. In the browser console the following appears (tried this in multiple environments and browsers)

### Errors and Stack Trace (if available)

```shell
express-forms/data:1 
Failed to load resource: the server responded with a status of 500 (Internal Server Error)

vendor.js?v=1715966999:2 TypeError: Cannot read properties of undefined (reading 'forms')
    at n.ImportExpressForms (client.js?v=1715966999:1:615933)
    at ja (vendor.js?v=1715966999:2:1083277)
    at Cu (vendor.js?v=1715966999:2:1094908)
    at js (vendor.js?v=1715966999:2:1140999)
    at wl (vendor.js?v=1715966999:2:1129284)
    at bl (vendor.js?v=1715966999:2:1129212)
    at ml (vendor.js?v=1715966999:2:1129075)
    at al (vendor.js?v=1715966999:2:1125882)
    at ll (vendor.js?v=1715966999:2:1126271)
    at Ho (vendor.js?v=1715966999:2:1067118)
```

### How can we reproduce this?

1. 
2. 
3. 

### Freeform Edition

Express

### Freeform Version

5.2.0

### Craft Version

4.9.3

### When did this issue start?

Fresh install of Freeform

**Comment by kjmartens** [comment]
Sorry about this @jmcgaha.

I can duplicate this in my testing as well. It appears to be a new issue that started in 5.1.19+. We'll get it fixed right away. 🙂

**Comment by kjmartens** [comment]
This is now fixed in Freeform 5.2.1+. 🙂
